src/hardware/vga_crtc.cpp

```diff
@@ -26,6 +26,12 @@
 #include "pic.h"
 #include "vga.h"
 
+static bool is_svga_or_vesa(VGAModes mode)
+{
+	return mode == M_LIN4 || mode == M_LIN8 || mode == M_LIN15 ||
+		mode == M_LIN16 || mode == M_LIN24 || mode == M_LIN32;
+}
+
```

Comment:
The C hacker in me kind of wants to change this to:

```c
return mode & (M_LIN4 | MLIN_8 | M_LIN15 | M_LIN16 | M_LIN24 | M_LIN32);
```
Since this enum is also a bitfield.  At the same, mis-reading what bitwise instructions are doing has been a problem.

Reply:
I figured you would say something like that :smile: